third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -961,11 +961,9 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		ResourceLabels: expandStringMap(d, "resource_labels"),
 	}
 
-<% unless version == 'ga' -%>
 	if v, ok := d.GetOk("default_max_pods_per_node"); ok {
 		cluster.DefaultMaxPodsConstraint = expandDefaultMaxPodsConstraint(v)
```

Comment:
The function called here will need to be made available as well.

https://github.com/GoogleCloudPlatform/magic-modules/blob/a3a434cf7b17e2b53f956539c175f52c06b66023/third_party/terraform/resources/resource_container_cluster.go.erb#L1209-L1211 and the `flatten` function as well.

And this test (I think all the other features are GA) + the config it links at the bottom of the test file. https://github.com/GoogleCloudPlatform/magic-modules/blob/a3a434cf7b17e2b53f956539c175f52c06b66023/third_party/terraform/tests/resource_container_cluster_test.go.erb#L1406